specs/signature-envelope-cose.md

```diff
@@ -36,12 +36,21 @@ Signature Manifest Example
     }
 }
 ```
+## Detached Signature Storage
```

Comment:
Technically oci signatures are detached signatures, can we say `Blob Signature Storage` ?